cou/steps/plan.py

```diff
@@ -113,6 +115,18 @@ async def generate_plan(analysis_result: Analysis, args: CLIargs) -> UpgradePlan
     return plan
 
 
+async def post_upgrade_sanity_checks(analysis_result: Analysis) -> None:
+    """Run post upgrade sanity checks.
+
+    :param analysis_result: Analysis result.
+    :type analysis_result: Analysis
+    """
+    try:
+        await verify_ceph_cluster_noout_unset(analysis_result.model)
+    except ApplicationError:
+        print("Upgrade completed, please unset noout for ceph cluster.")
```

Comment:
I think it's safe enough to unset `noout` after ceph-osds are upgraded sucessfully. Maybe this can be a post-upgrade step of osd and not a general post_upgrade

Reply:
the set or unset noout is done by the operator manually, so we only want to warn the user to do so.

Reply:
Added cli flag for set / unset ops